applets/google-maps/src/main.ts

```diff
@@ -0,0 +1,160 @@
+import { applets } from "@web-applets/sdk";
+import { Loader } from "@googlemaps/js-api-loader";
+
+type Place = {
+  placeId: string | undefined;
+  name: string | undefined;
+  icon: string | undefined;
+  iconBGColor: string | undefined;
+  address: string | undefined;
+  location: {
+    lat: number;
+    lng: number;
+  };
+  rating: number | undefined;
+  ratingTotal: number | undefined;
+};
+
+let map;
+let infoWindow;
+
+const loader = new Loader({
+  apiKey: import.meta.env.VITE_PUBLIC_MAPS_API_KEY,
+  version: "weekly",
+  libraries: ["maps", "places"],
+});
+
+async function initialiseMap() {
+  const mapOptions = {
+    center: { lat: 0, lng: 0 },
+    zoom: 1,
+    mapId: "DEMO_MAP_ID" /** @TODO: Do we need a real mapId */,
```

Comment:
The MapId is associated to the Google account, so I've used the temporary one for now (see: https://developers.google.com/maps/documentation/javascript/reference/map#Map.DEMO_MAP_ID)